pkg/v3/observation_test.go

```diff
@@ -577,7 +577,7 @@ func TestLargeObservationSize(t *testing.T) {
 		})
 	}
 	largePerformData := [10001]byte{}
-	for i := 0; i < ObservationPerformablesLimit; i++ {
```

Comment:
I'm not sure what to do here; should this test still add the maximum number of performables (which will always exceed the max observation length), or should we change the test to expect the max observation length to be exceeded?

Reply:
Expect the max observation length to be exceeded, the number of elements is dynamic anyway